app/styles/global.css

```diff
@@ -1,12 +1,10 @@
-@import url("https://fonts.googleapis.com/css2?family=Inter:wght@100..900&display=swap");
+@import url('https://fonts.googleapis.com/css2?family=Inter:wght@100..900&display=swap');
 
 @tailwind base;
 @tailwind components;
 @tailwind utilities;
 
-body {
-  font-family: "Inter", sans-serif;
-}
+
 
 @layer base {
   :root {
```

Comment:
You're not allowed to edit the global.css

You should have met one of the maintainers with a request of what you needed.

Reply:
i have made the changes